### PR TITLE
Add Email-Templates Management API endpoints

### PR DESCRIPTION
**Pull request by lbalmaceda**

### CODE REVIEW

File: auth0/v3/management/email_templates.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        telemetry (bool, optional): Enable or disable Telemetry
+            (defaults to True)
+    """
+
+    def __init__(self, domain, token, telemetry=True):
+        self.domain = domain
+        self.client = RestClient(jwt=token, telemetry=telemetry)
+
+    def _url(self, id=None):
+        url = 'https://%s/api/v2/email-templates' % self.domain
```
Comment (joshcanhelp) [inline auth0/v3/management/email_templates.py]: Just curious ... is this `sprintf`-type replacement more common than string interpolation in Python? Or is this just a convention for this SDK?

File: auth0/v3/management/email_templates.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        if id is not None:
+            return url + '/' + id
+        return url
+
+    def create(self, body):
+        """Create a new email template.
+
+        Args: 
+           body (dict): Attributes for the new email template.
```
Comment (joshcanhelp) [inline auth0/v3/management/email_templates.py]: Might be good to mention the required params. Saw your Jira note about this not being a convention here but could start with this endpoint? Not a show-stopper.

Comment (lbalmaceda) [reply auth0/v3/management/email_templates.py]: Also, I guess that's why the repo owner linked to the documentation whenever a `body` was present in the function params.

File: auth0/v3/management/email_templates.py (hunk 1 above)
Comment (lbalmaceda) [reply auth0/v3/management/email_templates.py]: I don't know the pros/cons. I'd have used regular string concatenation but I'm copy-pasting another entity basic constructor here, so I'd say this is "convention for this SDK" :trollface:

**Pull request merged**